Artiom/src/main/java/Retailer.java

```diff
@@ -0,0 +1,65 @@
+
+import java.util.*;
+
+public class Retailer {
+    public static void main(String[] args) {
+
+        //Create first order
+        Map<String, Address> firstUserAddress = new HashMap<>();
+        firstUserAddress.put("Home", new Address("Moldova", "Chisinau", "Stefan cel Mare"));
+        User firstUser = new User("Pikachu", firstUserAddress);
```

Comment:
What if you want to add one more address to this user? Or delete one of them? How can you handle this?

Reply:
Not bad, but why not adding one method in User class like **addAddress(Address address)** or **removeAddress(String key)**